shared/src/test/scala/mlscript/ReplHost.scala

```diff
@@ -0,0 +1,193 @@
+package mlscript
+
+import mlscript.utils.shorthands._
+
+/**
+ * A helper class manipulate an interactive Node.js process.
```

Comment:
Grammar is wrong. Did you mean this?
```suggestion
 * A helper class to manipulate an interactive Node.js process.
```